lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
We need to `require "action_controller"` before using it and doing so should not trigger any initialization. If it does, it is a bug in Rails.